### DISCUSSION TITLE
[General] Patch for fixed op-stack fee estimation support for ethers.js is now available

### DISCUSSION THREAD

**Discussion by ghost** [ethereum-optimism/developers#56] [General]
### Discussion Topic

Currently, ethers.js doesn't allow spending dynamic `maxPriorityFeePerGas` fetched from `eth_maxPriorityFeePerGas` which would lead users spending high fees to sequencers ( more than 1 Gwei loss! )

https://github.com/ethers-io/ethers.js/pull/4463/

With the patch being implemented, it will fix the compatibility issue with ethers.js ( Heard that viem has already fixed it ).

If you would like to see the patch being implemented for the optimism community, please show some support on the pull request.

**Comment by opfocus** [comment]
I think we need it

**Comment by ghost** [comment]
The feature is available since [6.9.0](https://github.com/ethers-io/ethers.js/releases/tag/v6.9.0)!
